libs/service-portal/finance/src/screens/FinanceStatus/FinanceStatus.tsx

```diff
@@ -114,33 +115,7 @@ const FinanceStatus: ServicePortalModuleComponent = () => {
               />
             </Box>
           )}
-          {!loading &&
-            !error &&
-            (financeStatusData?.organizations?.length === 0 ||
-              financeStatusData?.organizations === null) && (
-              <Box marginTop={2}>
-                <T.Table>
-                  <ExpandHeader
-                    data={[
-                      { value: formatMessage(m.feeCategory) },
-                      { value: formatMessage(m.guardian) },
-                      { value: formatMessage(m.status), align: 'right' },
-                    ]}
-                  />
-                  <T.Body>
-                    <ExpandRow
-                      last
-                      data={[
-                        { value: formatMessage(m.total) },
-                        { value: '' },
-                        { value: getChargeTypeTotal(), align: 'right' },
-                      ]}
-                    />
-                  </T.Body>
-                </T.Table>
-              </Box>
-            )}
-          {financeStatusData?.organizations?.length > 0 ? (
+          {financeStatusData?.organizations?.length > 0 || financeStatusZero ? (
```

Comment:
should pull this check out and resuse here and on line 196?
https://github.com/island-is/island.is/pull/5137/files#diff-822997789905148344f1668b55b36d844e1ba0830f38297858835b3a72415d22R196

Reply:
I believe it will not be needed there.

Because the _Total Row_ is outside of that check and it will show a row containing:

Samtals - 0kr. with the help of `getChargeTypeTotal` if there are no organizations present.